static/css/components/form.olform.less

```diff
@@ -84,16 +84,38 @@
     }
   }
 
+  .links-form {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    gap: 5px;
+    padding-bottom: 20px;
+
+    .links-form__inputs {
+      display: flex;
+      flex-flow: row wrap;
+      gap: 5px;
+      width: 100%;
+    }
+
+    .links-form__label, .links-form__url {
+      flex: 1 1 0;
+      min-width: 250px;
+    }
+  }
+
   input[type=number],
   input[type=text],
-  input[type=password] {
+  input[type=password],
+  input.addweb {
```

Comment:
I _think_ using `input[type=url]` might be ok here? We generally want all of these input fields to appear the same.

Reply:
Very possibly! Will mess around to see if that has unfortunate effects on other pages, and switch it if it seems safe. 😊